src/components/auth/AuthTorus.js

```diff
@@ -22,18 +22,41 @@ Image.prefetch(illustration)
 const log = logger.child({ from: 'AuthTorus' })
 
 const AuthTorus = ({ screenProps, navigation, styles, store }) => {
-  const torus = new TorusSdk({
-    typeOfLogin: 'google',
-    verifier: 'google-gooddollar',
-    GOOGLE_CLIENT_ID: config.googleClientId,
-    FACEBOOK_APP_ID: config.facebookAppId,
-    proxyContractAddress: '0x4023d2a0D330bF11426B12C6144Cfb96B7fa6183', // details for test net
-    network: 'ropsten', // details for test net
-    redirect_uri: `${config.publicUrl}/redirect.html`,
-  })
-
   const asGuest = true
+  const [serviceWorker, setServiceWorker] = useState(undefined)
+  const torus = useMemo(
+    () =>
+      new TorusSdk({
+        typeOfLogin: 'google',
+        verifier: 'google-gooddollar',
+        GOOGLE_CLIENT_ID: config.googleClientId,
+        FACEBOOK_APP_ID: config.facebookAppId,
+        proxyContractAddress: '0x4023d2a0D330bF11426B12C6144Cfb96B7fa6183', // details for test net
+        network: 'ropsten', // details for test net
+        redirect_uri: `${config.publicUrl}/torus/redirect`,
+        enableLogging: config.env === 'development',
+      }),
+    []
+  )
 
+  const registerTorusWorker = async () => {
+    try {
+      const registration = await navigator.serviceWorker.register(`${config.publicUrl}/torus//sw.js`)
```

Comment:
I think here is a typo in the path '.../torus//sw.js'. Should be '../torus/sw.js'

Reply:
fixed

Reply:
maybe this was the issue you had.
try again
also why isnt the window a popup?
please open the console when you take a video if it doesnt work